### PR TITLE
support solfuzz

### PR DESCRIPTION
**Pull request by nlgripto**

### CODE REVIEW

**Review by mjain-jump — DISMISSED** [review]

**Review by topointon-jump — DISMISSED** [review]

File: .github/workflows/clusterfuzz.yml [hunk 1]
```diff
@@ -50,3 +50,13 @@ jobs:
...
+
+      - name: make lib
+        run: unset MACHINE && EXTRAS=fuzz make -j -Otarget lib
+
+      - name: upload lib artifact
+        uses: actions/upload-artifact@v4
+        with:
+          path: ./build/native/clang/lib/libfd_exec_sol_compat.so
+          name: libfd_exec_sol_compat
+          retention-days: 14
```
Comment (lheeger-jump) [inline .github/workflows/clusterfuzz.yml:62]:
```suggestion
          retention-days: 14

```

File: .github/workflows/clusterfuzz.yml [hunk 2]
```diff
@@ -50,3 +50,13 @@ jobs:
...
           service-account-credentials: ${{ secrets.FUZZ_SERVICE_ACCT_JSON_BUNDLE }}
+
+      - name: make lib
+        run: unset MACHINE && EXTRAS=fuzz make -j -Otarget lib
+
+      - name: upload lib artifact
+        uses: actions/upload-artifact@v4
+        with:
+          path: ./build/native/clang/lib/libfd_exec_sol_compat.so
+          name: libfd_exec_sol_compat
```
Comment (lheeger-jump) [inline .github/workflows/clusterfuzz.yml:61]: does this need a name associated with a commit? Or will it have that

**Review by lheeger-jump — CHANGES_REQUESTED** [review]

File: .github/workflows/clusterfuzz.yml (hunk 2 above)
Comment (nlgripto) [reply .github/workflows/clusterfuzz.yml:61]: don't need it for my purposes

Comment (lheeger-jump) [reply .github/workflows/clusterfuzz.yml:61]: okay. will check on the first run of this

**Review by lheeger-jump — APPROVED** [review]

**Pull request merged**